### PR TITLE
added Dashboard layout

### PR DESCRIPTION
**Pull request by omjadhav85**
added design layout of dashboard page for Pomodoro mini app.

### CODE REVIEW

File: src/components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.modal-container {
+  /* display: none; */
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 99999999;
```
Comment (AnuragSahu11) [inline src/components/Modal/Modal.css]: you should not use such high z-index

**Review by AnuragSahu11 — APPROVED** [review]
In your deploy preview there was horizontal scroller, you may want to fix that, other than that , UI  
is good and minimalistic , good use of destructuring. Good to merge.

File: src/components/Modal/Modal.css (hunk 1 above)
Comment (omjadhav85) [reply src/components/Modal/Modal.css]: Thanks Anuraj for pointing that out. I have changed it a lower value.

**Review by Tanay-J — APPROVED** [review]
Good work! Liked how you used utility classes to avoid code repetition. Good coding practices followed as well. 
Approving the changes.

**Comment by omjadhav85** [comment]
Thanks Tanay and Anuraj for reviewing my PR! Merging after two reviews.

**Pull request merged**